src/org/zaproxy/zap/extension/bugtracker/BugTrackerBugzilla.java

```diff
@@ -0,0 +1,392 @@
+    private String FIELD_PLATFORM = "bugtracker.trackers.bugzilla.issue.platform";
+    private String FIELD_DESCRIPTION = "bugtracker.trackers.bugzilla.issue.description";
+    private String FIELD_SUMMARY = "bugtracker.trackers.bugzilla.issue.summary";
+    private String FIELD_USERNAME = "bugtracker.trackers.bugzilla.issue.username";
+    private String FIELD_PASSWORD = "bugtracker.trackers.bugzilla.issue.password";
+    private String FIELD_BUGZILLA_CONFIG = "bugtracker.trackers.bugzilla.issue.config";
+    private String summaryIssue = null;
+    private String descriptionIssue = null;
+    private JPanel configTable = null;
+    private JPanel bugzillaPanel = null;
+    private BugTrackerBugzillaTableModel bugzillaModel = null;
+    private RaiseSemiAutoIssueDialog dialog = null;
+
+    private static final Logger log = Logger.getLogger(BugTrackerBugzilla.class);
+
+    public void setDetails(Set<Alert> alerts) {
+        setSummary(alerts);
+        setDesc(alerts);
+    } 
+
+    public void setDialog(RaiseSemiAutoIssueDialog dialog) {
+        this.dialog = dialog;
+    }
+
+    public BugTrackerBugzilla() {
+        initializeConfigTable();
+    }
+
+    public void initializeConfigTable() {
+        bugzillaPanel = new BugTrackerBugzillaMultipleOptionsPanel(getBugzillaModel());
+    }
+
+    public JPanel getConfigPanel() {
+        return bugzillaPanel;
+    }
+
+    public void createDialogs() {
+        dialog.setXWeights(0.1D, 0.9D);
+        List<BugTrackerBugzillaConfigParams> configs = getOptions().getConfigs();
+        List<String> configNames = new ArrayList<String>();
+        for(BugTrackerBugzillaConfigParams config: configs) {
+            configNames.add(config.getName());
+        }
+        dialog.addComboField(FIELD_BUGZILLA_CONFIG, configNames, "");
+        dialog.addTextField(FIELD_URL, "");
+		dialog.addTextField(FIELD_PRODUCT, "");
+		dialog.addTextField(FIELD_COMPONENT, "");
+		dialog.addTextField(FIELD_VERSION, "");
+		dialog.addTextField(FIELD_OS, "");
+		dialog.addTextField(FIELD_PLATFORM, "");
+		dialog.addMultilineField(FIELD_DESCRIPTION, getDesc());
+		dialog.addTextField(FIELD_SUMMARY, getSummary());
+		dialog.addTextField(FIELD_USERNAME, "");
+		dialog.addTextField(FIELD_PASSWORD, "");
+	}
+
+	public void setSummary(Set<Alert> alerts) {
+		StringBuilder summary = new StringBuilder("");
+		for(Alert alert: alerts) {
+            if(alert.getName().length() > 0 ) {
+                summary.append(alert.getName().toString() + ", ");
+            }
+        }
+        summary.replace(summary.length()-2, summary.length()-1, "");
+        summaryIssue = summary.toString();
+	}
+
+	public void setDesc(Set<Alert> alerts) {
+		StringBuilder description = new StringBuilder("");
+		for(Alert alert: alerts) {
+
+            if(alert.getName().length() > 0 ) {
+                description.append(" *ALERT IN QUESTION* \n" + alert.getName().toString() + "\n\n");
```

Comment:
These strings should really be in the language file. Not mandatory for alpha addons, but def recommended.